test/test_core_notes.py

```diff
@@ -1,47 +1,188 @@
-#------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
What's changing in these comments?

Reply:
I tried to follow PEP8 to the letter as it was mentioned in the Hacking Guide.
https://www.python.org/dev/peps/pep-0008/#id31
PEP8 states that block comments should be a "# " (a hash symbol followed by a space) but the code didn't adhere to that so I changed it. I then removed a single "-" char from the end of the line to keep to the PEP8 line length https://www.python.org/dev/peps/pep-0008/#id19

Basically, the actions of a nervous first time contributor trying to stick to the letter of the law :)

Reply:
No worries.
Let's keep them consistent with the other header comments in pyelftools

Reply:
I've reverted the header style to the way it was previously